src/ocrd_network/utils.py

```diff
@@ -36,6 +39,24 @@ def calculate_execution_time(start: datetime, end: datetime) -> int:
     return int((end - start).total_seconds() * 1000)
 
 
+def convert_url_to_uds_format(url: str) -> str:
+    return f"http+unix://{url.replace('/', '%2F')}"
+
+
+def expand_page_ids(page_id: str) -> List:
```

Comment:
Move this to `ocrd_utils` and use it in `OcrdMets.get_physical_pages`, too?

Reply:
Sure, I could move it to `ocrd_utils`. However, I am not sure whether it is a good idea to move it inside `OcrdMets.get_physical_pages`. That method seems very complex. I would suggest splitting it few simpler methods and use `ge_physical_pages` as a wrapper over the simpler methods.